db/migration/1748446282000-DropSuggestedChartRevisionsTable.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class DropSuggestedChartRevisionsTable1748446282000
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            "DROP TABLE IF EXISTS `suggested_chart_revisions`"
+        )
+    }
+
+    public async down(_queryRunner: QueryRunner): Promise<void> {
+        // No rollback - table removal is permanent
```

Comment:
I just noticed this, and it's hopefully not really an issue, but as a reference for the future: proper migrations should always have a `down` migration, which is easy in this case: create the table.